integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/segment/ShowSegmentTestCase.scala

```diff
@@ -224,6 +224,37 @@ class ShowSegmentTestCase extends QueryTest with BeforeAndAfterAll {
     sql("drop table if exists a")
   }
 
+  test("test loading data into partitioned table with segment's updateDeltaEndTimestamp not change") {
```

Comment:
Thanks for working on it. Just a small suggestion. This showSegment testcase is meant for only segment display related test case. 

can you move this test case to existing `InsertIntoCarbonTableTestCase` file ?

Reply:
Thanks for reviewing, done.